scripts/superdesk-authoring/authoring.js

```diff
@@ -102,7 +102,7 @@
     }
 
     function stripHtml(item) {
-        var fields = ['headline', 'abstract'];
```

Comment:
Not in our case. Abstract field in front end is mapped to abstract field in the back end. If the html is not stripped, backend needs to change to strip it there.